documentation-website/Writerside/topics/SQL-Functions.md

```diff
@@ -88,9 +105,9 @@ val replacedName = CustomFunction<String?>("REPLACE", VarCharColumnType(), FooTa
 `CustomFunction` class accepts a function name as a first parameter and the resulting column type as second. After that, you can provide any amount of parameters separated by a comma.
 
 There are also shortcuts for string, long, and datetime functions:
-* `CustomStringFunction`
-* `CustomLongFunction`
-* `CustomDateTimeFunction`
```

Comment:
Could `CustomFunction` above (line 105) also get an api link? Can't seem to find one anywhere.